FusionIIIT/templates/academic_procedures/allot_courses.html

```diff
@@ -0,0 +1,133 @@
+{% load static %}
+{% block allot_courses %}
+
+{% if messages %}
+<ul class="messages">
+    {% for message in messages %}
+    <h2>
+    <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+    </h2>
+    {% endfor %}
+</ul>
+{% endif %}
+
+<div class="ui pointing secondary menu">
+    <a class="active item" data-tab="allotment">
+        Allot Student Courses
+    </a>
+</div>
+
+
+
+<div class="ui active tab segment" data-tab="allotment">
+
+    <form class="ui form" method="POST" action='/academic-procedures/acad_person/allot_courses/'
+        enctype="multipart/form-data" id='allot_courses_form'>
+        {% csrf_token %}
+        <div class="field">
+            <h4 class="ui blue header">Note : Provide the data in Excel Sheet in following format:</h4>
+            RollNo | CourseSlot Name | CourseCode of alloted course
+            <h4 class="ui dividing header"><a href="../media/Administrator/academic_information/sheet.xlsx">Download</a>
```

Comment:
I don't know, how the "Download" link will work, you haven't pushed anything...

Reply:
Media folder is gitignored so I cannot add excel in that directory, but it is working.